views/main.ejs

```diff
@@ -0,0 +1,10 @@
+<h3>Main Page Yo!</h3>
+
+<form action="/urlShow" method="POST">
+  <div class="form-group">
+    <input type="text" name="url" class="form-control" placeholder="URL" required>
```

Comment:
As I mentioned res.redirect needs http:// or it wonr redirect to the address. By setting the input type to url instead of text it does a front end validation and alerts the user that the address is invalid. type="url"